src/components/pages/itemDetail/itemDetail.jsx

```diff
@@ -0,0 +1,414 @@
+                            </div>
+                    </div>
+                    <div css={itemUserInfo}>
+                        <div css={userImg}>
+                            <img src={userImage} />
+                        </div>
+                        <div css={userText}>
+                            <h4>{item.ownerNickname}</h4>
+                            <p>{new Date(item.createdAt).toISOString().slice(0, 10)}</p>
+                        </div>
+                        <div css={likeSection}>
+                            <button>💛</button>
+                            {item.favoriteCount}
+                        </div>
+                    </div>
+                </div>
+            
+            </div>
+
+            <div css={commentSection}>
+                <div css={makeComment}>
+                    <label htmlFor="makecomment">문의하기</label>
+                    <textarea 
+                        id="makecomment" 
+                        placeholder="개인정보를 공유 및 요청하거나, 명예 훼손, 무단 광고, 불법 정보 유포시 모니터링 후 삭제될 수 있으며, 이에 대한 민형사상 책임은 게시자에게 있습니다."
+                        value={commentTextarea}
+                        onChange={handleChange}
+                        />
+                    <button type="button" css={[commentButtonStyle, isButtonActive && activeButtonStyle]} >등록</button>
+                </div>
+                <ul css={commentList}>
+                    {comments.length == 0 ? 
+                    <div>
+                        <img src={noCommentImg} css={noCommentImgStyles} />
+                    </div>
+                    :
+                        comments.map((comment) =>(
+                            <li key={comment.id} css={commentListStyle}>
+                                <div css={commentHeader}>
+                                    <p>{comment.content}</p>
+                                    <img src={kebabIcon} css={editIconStyle} />
+                                </div>
+                                
+                                <div css={itemUserInfo}>
+                                    <div css={userImg}>
+                                        <img src={userImage} />
+                                    </div>
+                                    <div css={userText}>
+                                        <h4>{comment.writer.nickname}</h4>
+                                        <p>{new Date(comment.createdAt).toISOString().slice(0, 10)}</p>
+                                    </div>
+                                    <hr />
+                                </div>
+                            </li>
+                        ))
+                    }                
+                </ul>
+            </div>
+            <Link to={"/items"} css={linkStyle}>
+                <button type='button' css={backToListButtonStyle} >
+                    <p>목록으로 돌아가기</p>
+                    <img src={backIcon} />
+                </button>
+            </Link>
```

Comment:
💊 제안
단순 페이지 이동이니 Link 태그를 사용하시는 것이 더 적절해보입니다. 
디자인이 버튼처럼 보이신다고 꼭 버튼 태그를 사용하실 필요는 없습니다. Link 태그에 css로 디자인대로 스타일링해주세요~
```suggestion
            <Link to={"/items"} css={[linkStyle,backToListButtonStyle]}>
                  <p>목록으로 돌아가기</p>
                  <img src={backIcon} />
            </Link>
```